src/headers/game.hpp

```diff
@@ -16,6 +16,7 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <cstdio>
```

Comment:
If the snprintf() function is replaced with its C++ version, there is no need for this header include.